packages/GA4Client/test/src/tests.js

```diff
@@ -3022,14 +3050,14 @@ describe('Google Analytics 4 Event', function () {
 
             // Initial elements of Data Layer are setup for gtag.
             // Consent Default is index 3
-            // Consent Update is index 4
-            // Event is index 5
-            // Consent Update #2 is index 6
-            // Event #2 is index 7
-            window.dataLayer.length.should.eql(8);
-            window.dataLayer[6][0].should.equal('consent');
-            window.dataLayer[6][1].should.equal('update');
-            window.dataLayer[6][2].should.deepEqual(expectedDataLayerFinal[2]);
+            // Consent Update is index 5
```

Comment:
what is at index 4?

Reply:
I believe its the 'Test Event' event, will console.log it, confirm and add it for reference

Reply:
oh wait nvm, its the initial update since the new workflow doesn't merge defaults and fire two consent payloads on init (one default one update)